recipes/r-bigsnpr/meta.yaml

```
@@ -52,15 +55,22 @@ requirements:
     - r-foreach
     - r-ggplot2
     - r-magrittr
+    - r-quadprog
     - r-rmio
+    - r-rmutil 
     - r-roptim >=0.1.6
+    - r-rsqlite
```

Move to `test` only.

```suggestion
    - r-rmio
    - r-roptim >=0.1.6
```